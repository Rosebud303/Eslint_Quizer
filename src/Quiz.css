.quiz-image {
    width: 100%;
    background: #282a22;
    padding: 1em;
    box-sizing: border-box;
    border-radius: 5px;
}

.quiz-title {
    text-align: center;
}