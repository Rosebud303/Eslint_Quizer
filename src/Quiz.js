import React from 'react';
import './Quiz.css'

export default function Quiz(prop) {
  return (
    <div>
      <h2 className='quiz-title'>Choose a rule!</h2>
      <img className='quiz-image' src={prop.current.quiz}/>
    </div>
  )
}
