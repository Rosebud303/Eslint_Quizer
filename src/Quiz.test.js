import React from 'react';
import Quiz from './Quiz.js';
import { shallow } from 'enzyme';

const mockQuest ={
    "name": "Key Spacing",
    "quiz": "https://i.imgur.com/6F1XUXC.png",
    "answer": "https://i.imgur.com/gcQ23xw.png",
    "id": 4
}

describe('Quiz', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <Quiz current={mockQuest}/>
        )
    });

    it('Should match the snapshot' , () => {
        expect(wrapper).toMatchSnapshot();
    });
});