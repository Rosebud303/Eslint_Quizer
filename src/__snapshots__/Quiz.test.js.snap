// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Should match the snapshot 1`] = `
<div>
  <h2
    className="quiz-title"
  >
    Choose a rule!
  </h2>
  <img
    className="quiz-image"
    src="https://i.imgur.com/6F1XUXC.png"
  />
</div>
`;
